cart/application/cartService.go

```diff
@@ -1013,3 +1037,14 @@ func (cs *CartService) getProductWithActiveVariantIfProductIsConfigurable(ctx co
 
 	return product, nil
 }
+
+// HasRemovedCouponCodes returns if a QtyAdjustmentResults has any adjustment that removed a coupon code from the cart
+func (qar QtyAdjustmentResults) HasRemovedCouponCodes() bool {
```

Comment:
If updating the struct property maybe also changing the function name.

Reply:
See above.